lib/vmpooler/api.rb

```diff
@@ -201,6 +203,8 @@ def hostname_shorten hostname
             result['clone_average'] = $redis.hvals('vmpooler__clone__'+Date.today.to_s).map( &:to_f ).reduce( :+ ) / result['clone_total']
           end
 
+          result['uptime'] = Time.now - $config[:uptime]
```

Comment:
Yes, 'clone_average' and now 'status' are both in seconds.  I'll keep in mind to maintain consistency throughout this endpoint.